libphobos/configure.ac

```diff
@@ -107,7 +109,13 @@ fi
 d_target_os=`echo $target_os | sed 's/^\([A-Za-z_]+\)/\1/'`
 
 GDC=$CC
-GDC=`echo $CC | sed s/xgcc/gdc/`
+# FIXME: This is not really reliable. CC can include complete paths,
+# arguments, etc which may contain 'gcc'
+if test -z "${with_target_subdir}"; then
```

Comment:
Use ``test -n``.

And yes, I think it might be better if we instead patch the toplevel ``Makefile.tpl`` and ``configure.ac`` to export ``DC`` and ``DFLAGS`` for us.